examples/fabric-bridge-app/linux/main.cpp

```diff
@@ -81,9 +87,18 @@ void BridgePollingThread()
             int ch = getchar();
             if (ch == 'e')
             {
-                ChipLogProgress(DeviceLayer, "Exiting.....");
+                ChipLogProgress(NotSpecified, "Exiting.....");
                 exit(0);
             }
+#if defined(PW_RPC_FABRIC_BRIDGE_SERVICE)
```

Comment:
What is `defined(PW_RPC_FABRIC_BRIDGE_SERVICE) == 0`? do we still want this? same for all other instances of this `#if defined`

Reply:
yes, we should use #if defined(PW_RPC_FABRIC_BRIDGE_SERVICE) && PW_RPC_FABRIC_BRIDGE_SERVICE as other example apps.